recipes/daggy/2.1/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from conans import ConanFile, CMake, tools, RunEnvironment
+
+
+class DaggyTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_paths", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            os.chdir("bin")
+            self.run(".%stestcpp" % os.sep, run_environment=True)
+            self.run(".%stestc" % os.sep, run_environment=True)
```

Comment:
This is our usual, easier to read

```suggestion
            self.run(os.path.join("bin", "testcpp"), run_environment=True)
            self.run(os.path.join("bin", "testc"), run_environment=True)
```